apis/slo/v1alpha1/common.go

```diff
@@ -0,0 +1,28 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	schedulingv1alpha1 "github.com/koordinator-sh/koordinator/apis/scheduling/v1alpha1"
+)
+
+type ResourceMap struct {
+	corev1.ResourceList `json:"resources,omitempty"`
+	Devices             []schedulingv1alpha1.DeviceInfo `json:"devices,omitempty"`
```

Comment:
for usage report? `total capacity`comments of DeviceInfo.Resource at "https://github.com/koordinator-sh/koordinator/blob/91cacc405bb286a20d0af4f921ca779f867faded/apis/scheduling/v1alpha1/device_types.go#L45" needs modify